### ISSUE TITLE
Requests: Please decrease the font size and show more text by default

### ISSUE THREAD

**Issue by mschnitzer** [openSUSE/open-build-service#7056]
**Is your feature request related to a problem? Please describe.**
The request page on the new WebUI shows only one line of the description. It would be good to see at least 5-10. Maybe decreasing the font size would also be better when you increase the lines that will be shown.

//Edit: Actually, it seems that the default is set to more than one line, but this doesn't work for every request it seems. So I got a request, that shows more than one line, but in parallel, I've opened a second request that only shows one line. Is this maybe a bug?

**Describe the solution you'd like**
More lines are visible by default.

**Describe alternatives you've considered**
n/a

**Additional context**
Reach out to me to receive the links of both requests

**Comment by bgeuken** [comment]
I added some code so that we also consider the new lines when deciding if a collapsible is needed. That should cause the whole text being visible in more cases.
For showing more than one line when the text is collapsed we probably would have to replace the whole solution (using CSS / eclipse) we currently use.

**Comment by bgeuken** [comment]
Closing since the main issue was solved. To solve the problem that only one line is show when in collapsed mode we will have to re-write the collapsible plugin. That's out of scope of this issue.
